src/modules/displayComment.js

```diff
@@ -1,17 +1,21 @@
 const displayComment = (meal) => {
-  console.log(meal)
-  // const commentModal = document.createElement('div');
-  // commentModal.className='modal-popup'
-  // const mealObj=meal.meals[0];
-  // commentModal.innerHTML=`
-  // <img src="${mealObj.strMealThumb}" alt="Meal image">
-  // <h1>${mealObj.strMeal}</h1>
-  // <p>${mealobj.strInstructions}</p>
-  // <h3>Commets</h3>
-  // <p> ...list of comments</p>
-  // <h3>Add a Comment>
-  // <input type="text" id='comment-mame' placeholder="Your Name">
-  // <input type="text" id='comment-content' placeholder="Your Insights">
-  // `
-}
+  meal.then((mealObj) => {
+    const selectMeal = mealObj.meals[0];
+    console.log(mealObj)
+    const commentModal = document.createElement('div');
+    commentModal.className='modal-popup'
+    commentModal.innerHTML=`
```

Comment:
Your pop-up should have a unique ID so it is easily referenced in the js file. This will avoid any conflicts in larger projects.